client/my-sites/plans/jetpack-plans/product-store/hooks/use-item-lightbox.ts

```diff
@@ -0,0 +1,24 @@
+import { isEnabled } from '@automattic/calypso-config';
+import { useState } from 'react';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { SelectorProduct } from '../../types';
+
+export const useItemLightbox = () => {
+	const [ currentItem, setCurrentItem ] = useState< SelectorProduct | null >( null );
+	const clickMoreHandlerFactory = ( item: SelectorProduct ): VoidFunction | undefined => {
+		return isEnabled( 'jetpack/pricing-page-product-lightbox' )
```

Comment:
The lightbox in the new pricing rework page does not need to be behind the lightbox feature-flag.  The lightbox feature-flag is/was only for displaying the lightbox on the current (old) pricing page. (which I don't think we are going to do now, but we were planning on it in earlier stages).
So this lightbox feature-flag check can bee removed.

Reply:
Also, can we wrap the function in `useCallback`

```ts
const clickMoreHandlerFactory = useCallback( ( item: SelectorProduct ) => {
	return () => {
		recordTracksEvent( 'calypso_product_more_about_product_click', {
			product: item.productSlug,
		} );
		setCurrentItem( item );
	};
}, [] );
```

Reply:
thanks for catching this one, I agree useCallback is good here. thanks 👍